packages/admin-stories/src/admin/snackbar/CustomSnackbar.tsx

```diff
@@ -0,0 +1,78 @@
+import { SnackbarProvider, useSnackbarApi } from "@comet/admin";
+import { Button, List, ListItem, Snackbar } from "@material-ui/core";
+import { storiesOf } from "@storybook/react";
+import * as React from "react";
+import styled from "styled-components";
+
+const ShowSnackbarButton = styled(Button)`
+    && {
+        background-color: darkred;
+        color: white;
+    }
+
+    &&:hover {
+        opacity: 0.7;
+        background-color: darkred;
+    }
+`;
+
+const HideSnackbarButton = styled(Button)`
+    && {
+        background-color: blue;
+        color: white;
+
+        &&:hover {
+            opacity: 0.7;
+            background-color: blue;
+        }
+    }
+`;
+
+let counter = 0;
+
+const CustomSnackbar = () => {
+    const snackbarApi = useSnackbarApi();
+
+    const handleActionButtonClick = () => {
+        console.log("Do stuff here");
+        snackbarApi.hideSnackbar();
+    };
+
+    const showCustomSnackbar = () => {
+        snackbarApi.showSnackbar(
+            <Snackbar
+                anchorOrigin={{ vertical: "bottom", horizontal: "left" }}
+                // Use uuid or object id in production
+                key={counter++}
```

Comment:
Do you need to use a key here?

Reply:
Yes the key is needed. Otherwise the snackbar is updated in-place and there is no animation for indicating that the content changed